pipeline/src/graph-queries/addressables/exhibitionTexts.ts

```diff
@@ -2,6 +2,15 @@ const query = `
   exhibition-texts {
     title
     intro_text
+    related_exhibition {
```

Comment:
had to add this to the query because the `intro_text` is often empty, and if it is, we show the `related_exhibition`'s promo caption so we have to fetch that too